source/win-preferences/schema/advanced.js

```diff
@@ -35,6 +35,11 @@ export default {
         type: 'checkbox',
         label: trans('dialog.preferences.delete_on_fail'),
         model: 'system.deleteOnFail'
+      },
+      {
+        type:  'checkbox',
```

Comment:
Fails `yarn lint`: `40:13  error  Extra space before value for key 'type'  key-spacing`